### DISCUSSION TITLE
VS 2022 + "Avalonia for Visual Studio"?

### DISCUSSION THREAD

**Discussion by ToolmakerSteve** [AvaloniaUI/Avalonia#6890] [Q&A]
Today I downloaded the released VS 2022 Community. Found "Avalonia for Visual Studio" in Windows Store. Downloaded & ran .msix.

Only showed VS 2019 as an install option. (I also have that installed.)

Should I just stay with VS 2019 for now?

**Comment by Triky313** [comment]
I would stay for 2019, there are already some issues about it.
After installing VS2022, I was able to start my application, but of course there is no window preview.

https://github.com/AvaloniaUI/AvaloniaVS/issues/206
https://github.com/AvaloniaUI/AvaloniaVS/issues/222
